examples/src/main/java/io/grpc/examples/errorhandling/ErrorHandlingClient.java

```diff
@@ -67,11 +67,10 @@ public static void main(String [] args) throws Exception {
     new ErrorHandlingClient().run();
   }
 
-  private Server server;
   private ManagedChannel channel;
 
   void run() throws Exception {
-    server = ServerBuilder.forPort(0).addService(new GreeterGrpc.GreeterImplBase() {
+    Server server = ServerBuilder.forPort(0).addService(new GreeterGrpc.GreeterImplBase() {
```

Comment:
- will + // comment
- good questions. no idea... maybe it can be redesigned later but not in this PR.
- I think it does not need to specify a fixed port number because the client is in the same application and can get server's port number easily.